Majasdarbs/04-majasdarbs/bootstrap/04-katrina-tiltina/04-KT-CV-adaptivs-dizains/style.css

```diff
@@ -0,0 +1,438 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;1,300&display=swap');
+
+
+*{
+    margin:0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html{
+    font-size: 62.5%;
+    font-family: "Montserrat", sans-serif;
+    scroll-behavior: smooth;
+}
+
+a{
+    text-decoration: none;
+    color: white;
+}
+
+.main-header{
+    background: rgb(60, 22, 66);
+    width: 100%;
+    position: fixed;
+    z-index: 10;
+
+}
+
+h2{
+    font-size: 2.4rem;
+}
+
+h4{
+    font-size: 2.5rem;
+}
+
+h3{
+    font-size: 2rem;
+}
+
+p{
+    font-family: sans-serif;
+}
+
+li{
+    list-style: none;
+    font-size: 2rem;
+    
+}
+
+.list-style li{
+        margin-bottom: 1rem;
+        background: url(./icons/redo-arrow-1_icon-icons.com_56892.svg) no-repeat left top;
+        padding: 0px 0 3px 35px;
+}
+
+.nav{
+    width: 90%;
+    margin: auto;
+    display:flex;
+    align-items: center;
+    min-height: 10vh;
+    padding: 2rem 0;   
+}
+
+.logo{
+    flex: 1 1 40px;
+    color: aliceblue;
+    font-size: 3rem; 
+    text-transform: uppercase;
+
+}
+
+.nav-links{
+    list-style: none;
+    flex: 2 1 40px;
+    display: flex;
+    font-size: 2rem;
+    justify-content: space-around;
+}
+
+a.active{
+    color:rgb(175, 252, 65);
+}
+
+a:hover,a.active{
+    background: none;
+    color:rgb(175, 252, 65);
+    transition: .5s;
+}
+
+p{
+    font-size: 2rem;
+}
+/*main part starts here*/
+
+main{
+    height: 85vh;
+    background-image:linear-gradient(rgba(60, 22, 66,0.8), transparent), url("./img/3.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    color: white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start; 
+}
+
+.main-note{
+    margin-top: 10vh;
+    margin-left: 10%;
+    margin-right: 10%;
+    
+}
+
+.main-text{
+    font-size: 3rem;
+}
+
+button{
+    font-family: "Montserrat", sans-serif;
+    font-size: 2rem;
+    color: rgb(60, 22, 66);
+    padding: 9px 50px;
+    background-color: rgba(175, 252, 65);
+    border: none;
+    border-radius: 50px;
+    cursor: pointer;
+    transition: all 0.3s ease 0s;
+    margin-top: 30px; 
+}
+
+button:hover{
+    background-color: rgba(8, 99, 117,0.8);
+    color: rgb(175, 252, 65);
+    
+}
+
+/*second part starts here*/
+.benefits-head{
+    font-family: "Montserrat", sans-serif;
+    padding: 3rem;
+    min-height: 30vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    color:rgb(60, 22, 66);
+}
+
+.benefits-head h3{
+    padding: 1rem;
+    font-size: 2rem;
+    font-family:  sans-serif;
+    font-weight: lighter;
+    
+}
+.benefits-head h2{
+    font-family: "Montserrat", sans-serif;
```

Comment:
I think font-family is already inherited from the root, unless you override it somewhere